dali/base/dafdesc.cpp

```diff
@@ -2336,7 +2428,7 @@ IFileDescriptor *createFileDescriptor(const char *lname,IGroup *grp,IPropertyTre
         width = grp->ordinality();
     StringBuffer s;
     for (unsigned i=0;i<width;i++) {
-        makePhysicalPartName(lname, i+1, width, s.clear(), 0, os, nullptr, false);
+        makePhysicalPartName(lname, i+1, width, s.clear(), 0, os, nullptr, false, 0);
```

Comment:
Is 0 for the stripe correct?  Would be worth having a comment why. (added later?)